.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Build Bukkit plugin
+
+on:
+  push:
+    branches: [ master ]
```

Comment:
Master~

Move to main eventually:tm: or na?

Reply:
No idea - will have to be discussed at some point.

Reply:
> Master~

goddddd